### PR TITLE
JITM: hide message 3 seconds after clicking the module act. CTA

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

This will address part of #9462

I picked 3 seconds to give folks time to see that the button changed to an "Activated" status.

screen recording 2018-12-12 at 05 49 pm

#### Testing instructions:

* Go to Jetpack > Settings and deactivate the Asset CDN.
* Refresh the page; you should see a JITM offering you to activate the feature.
* Click activate, and notice the message going through the different phases of the process (activating, activated), before to disappear.

#### Proposed changelog entry for your changes:
* Dashboard notices: automatically dismiss notices once a feature has been activated.

### CODE REVIEW

**Review by keoshi — DISMISSED** [review]
Works great! Did you try with a `fadeOut`? It still causes the jump, but could be a neater way to make it go away. Just asking for curiosity, this looks good to me.

**Comment by keoshi** [comment]
Not related to this PR, but I noticed that the site accelerator settings are not updated on-the-fly when I click this button. They still show up deactivated unless I refresh the page.

**Comment by jeherve** [comment]
I believe they should update as well after a few seconds.

**Comment by jeherve** [comment]
> Did you try with a fadeOut? 

I'll give it a try now.

**Comment by jeherve** [comment]
@keoshi Here is how it looks after 814c0a4f7cf09187f30b25a2d4aab49693eb4437

screen recording 2018-12-13 at 01 09 pm

Is it better?

**Comment by keoshi** [comment]
I kinda dig it like this, what do you think? Could even use less of a timeout since it now has the animation, but looks great!

**Comment by jeherve** [comment]
I like it too. 

> Could even use less of a timeout since it now has the animation, but looks great!

Should it be 2 seconds instead of 3 then? I can make that change if needed.

**Comment by keoshi** [comment]
@jeherve I believe the `slow` indicates 600 milliseconds, so 2 seconds sounds great to me!

**Review by oskosk — DISMISSED** [review]
LGTM!

**Comment by oskosk** [comment]
I approved this PR but not quite clear if it's good to go with 3 seconds or 2.

**Comment by jeherve** [comment]
I changed it to 2 seconds in cf595242b2066b7fd3838077d322f378ff314dce.

@oskosk I'll let you merge if you like it :)

**Review by oskosk — APPROVED** [review]

**Pull request merged**
